### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by rahulyadav139**
Demo: https://gifted-leakey-419629.netlify.app/

### CODE REVIEW

File: src/components/listing-page/filters/FilterByCarModels.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { Fragment } from 'react';
+import './FilterByCarModels.css';
+import { v4 as uuid } from 'uuid';
+
+const FilterByCarModels = ({ products }) => {
+  const carDetails = products
+    .map(el => el.car)
+    .filter(el => el !== 'universal');
+  // // console.log([...carDetails]);
+  // const carDetailsUnique = new Set([...carDetails]);
```
Comment (dev-ashishsethi) [inline src/components/listing-page/filters/FilterByCarModels.js:10]: avoid comments please

File: src/components/listing-page/filters/FilterByPrice.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+      <input
+        min="1000"
+        max="10000"
+        step="1000"
+        class="styled-slider"
+        type="range"
+      />
+
+      <div class="hr-line fad"></div>
```
Comment (dev-ashishsethi) [inline src/components/listing-page/filters/FilterByPrice.js:28]: you could have used hr tag instead

**Review by dev-ashishsethi — APPROVED** [review]

File: src/components/listing-page/filters/FilterByCarModels.js (hunk 1 above)
Comment (manojsarna) [inline src/components/listing-page/filters/FilterByCarModels.js:10]: Suggestion : Keep it as clean as possible

File: src/components/listing-page/filters/FilterByRatings.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+const FilterByRatings = props => {
+  return (
+    <Fragment>
+      <h4>Rating</h4>
+
+      <div class="rating-wrapper">
+        {Array.from({ length: 4 }).map((el, i) => (
+          <div key={uuid()}>
+            <input
+              type="radio"
```
Comment (manojsarna) [inline src/components/listing-page/filters/FilterByRatings.js:14]: Can you elaborate on this?

File: src/components/listing-page/listing/Listing.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        <ul>
+          <li>1</li>
+          <li>2</li>
+          <li>3</li>
+          <li>4</li>
+          <li>5</li>
+          <li>{'>'}</li>
+        </ul>
+      </div>
+    </div>
```
Comment (manojsarna) [inline src/components/listing-page/listing/Listing.js:41]: Suggestion : Dont hard code values

File: src/components/listing-page/listing/Listing.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+          <i class="bi bi-filter-right"></i>Filter
+        </button>
+      </div>
+
+      <div class="sorting-container flex gap align-center">
+        <div class="heading-6">Sort By:</div>
+        {/* <input type="radio" name="sort" id="popularity" />
+        <label htmlFor="popularity">Popularity</label> */}
+        <input type="radio" name="sort" id="low-to-high" />
+        <label htmlFor="low-to-high">Price - Low to High</label>
```
Comment (manojsarna) [inline src/components/listing-page/listing/Listing.js:20]: Check for comments here too

**Review by manojsarna — CHANGES_REQUESTED** [review]
Overall App is Superb

Suggestions : 

1. Do Add Guest Login
2. Auth Error Handling Not Done
3. Signup Not Working
4. Show Password Not Working
5. No Loader Added

Check My Comments

**Review by manojsarna — APPROVED** [review]
Overall App is Superb

Suggestions : 

1. Do Add Guest Login
2. Auth Error Handling Not Done
3. Signup Not Working
4. Show Password Not Working
5. No Loader Added

Check My Comments

**Pull request merged**